composer.json

```diff
@@ -38,7 +38,7 @@
 	],
 	"require"    : {
 		"php"                      : ">=5.4",
-		"squizlabs/php_codesniffer": "^3.3.1",
+		"squizlabs/php_codesniffer": "^3.4.1",
```

Comment:
Any particular reason to change the minimum requirement ? And if so, why to this particular version ?

Reply:
I think this was the latest version when I started to work on it initially. I can up this to `^3.4.2`.

Reply:
But why change the minimum version at all at this time ? Is there anything in the newer PHPCS which this PR needs ? which TRTCS needs ? If not,why not keep it as, in line with the WPCS minimum requirements ?

Reply:
Ok, makes sense 🙂It's just a habit of mine to keep the libraries up to date (watching for BC breaks ofc).

Reply:
Not a bad habit at all, but that update would be unrelated to this PR, so it ought to be a separate PR in that case (which would also update the readme + the travis script to match the changed dependency requirements)